streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -133,6 +139,40 @@ private void createTopics() throws InterruptedException {
         CLUSTER.createTopics(outputTopic, outputTopicConcurrent, outputTopicConcurrentWindowed, outputTopicThree);
     }
 
+    /**
+     * Try to read inputValues from {@code resources/QueryableStateIntegrationTest/inputValues.txt}, which might be useful
+     * for larger scale testing. In case of exception, for instance if no such file can be read, return a small list
+     * which satisfies all the prerequisites of the tests.
+     */
+    private List<String> getInputValues() {
+        List<String> input = new ArrayList<>();
+        ClassLoader classLoader = getClass().getClassLoader();
+        String fileName = "QueryableStateIntegrationTest/inputValues.txt";
+        try (BufferedReader reader = new BufferedReader(new FileReader(classLoader.getResource(fileName).getFile()))) {
+            for (String line = reader.readLine(); line != null; line = reader.readLine()) {
+                input.add(line);
+            }
+        } catch (Exception e) {
+            log.warn("Unable to read '{}'. Using default inputValues list", "resources/" + fileName);
```

Comment:
as above: avoid `/`

Update to
```
log.warn("Unable to read '{}{}{}'. Using default inputValues list", "resources", File.seperator, fileName);
```

Reply:
Yes, that's certainly safer. Thanks. I also changed it in fileName and added `final` as requested.